backup/statistics.go

```diff
@@ -166,7 +166,7 @@ func generateAttributeSlotsQuery7(attStat AttributeStatistic) string {
 			AnyValues(attStat.Values2, attStat.Type),
 			AnyValues(attStat.Values3, attStat.Type),
 			AnyValues(attStat.Values4, attStat.Type),
-			AnyValues(attStat.Values5, attStat.Type))
+			ByteaValues(attStat.Values5))
```

Comment:
It's not clear for me (maybe my previous suggestion was wrong).

On the gpdb-5x there is only one `kind=99` - `HLL` and stored at [stavalues-4](https://github.com/greenplum-db/gpdb/blob/5X_STABLE/src/include/catalog/pg_statistic.h#L348-L353)
On the gpdb-6.x there are two kinds of HLL (full and not) kinds (98, 99) - them are stored at [stavalues4 too](https://github.com/greenplum-db/gpdb/blob/6X_STABLE/src/include/catalog/pg_statistic.h#L303-L316) and main branch has [such description too](https://github.com/greenplum-db/gpdb/blob/main/src/include/catalog/pg_statistic.h#L282-L295)

Reply:
6x and 7x have wrong description: it must `stavalues5`

Reply:
There is a thread at internal-core which describes this difference. But seems it should be commented at code or, described at the PR that HLL stores at different `stavalues` on 5.x and 6.x/7.x

Reply:
comments was added